src/main/java/frc/robot/subsystems/ArmSystem.java

```diff
@@ -23,17 +32,33 @@ public ArmSystem() {
 
         motor.restoreFactoryDefaults();
 
+        setUpShuffleboard();
+    }
+
+    private void setUpShuffleboard() {
```

Comment:
place `private` method below all the `public` ones

Reply:
My reasoning for ordering functions that way is that functions closer to other functions are behaviorally similar or in the same category. I'll put it below the `public` methods if that's the code style.